eng/perf-job.yml

```diff
@@ -0,0 +1,75 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osIdentifier: ''
+  readyToRun: true
+  helixQueues: ''
+  container: ''
+  crossrootfsDir: ''
+  framework: netcoreapp5.0 # Specify the appropriate framework when running release branches (ie netcoreapp3.0 for release/3.0)
+
+### Test job
+
+### Each test job depends on a corresponding build job with the same
+### buildConfig and archType.
+
+jobs:
+- template: /eng/common/templates/job/performance.yml
```

Comment:
We should only be running performance runs on Release builds. I don't know if we want to state that in the template or at instantiation time.

Reply:
I see it is in the `perf.yml`. - Thanks.